kornia/geometry/epipolar/numeric.py

```diff
@@ -24,3 +24,613 @@ def cross_product_matrix(x: torch.Tensor) -> torch.Tensor:
     cross_product_matrix_flat = torch.stack([zeros, -x2, x1, x2, zeros, -x0, -x1, x0, zeros], dim=-1)
     shape_ = x.shape[:-1] + (3, 3)
     return cross_product_matrix_flat.view(*shape_)
+
+
+# Reference
+# Nistér, David. An efficient solution to the five-point relative pose problem. 2004.
+# https://github.com/danini/graph-cut-ransac/blob/master/src/pygcransac/include/
+# estimators/solver_essential_matrix_five_point_nister.h#L108
+
+
+def o1(a: torch.Tensor, b: torch.Tensor) -> torch.Tensor:
```

Comment:
i think all this should be part of the recent `kornia.geometry.solvers` submodule
https://github.com/kornia/kornia/blob/master/kornia/geometry/solvers/polynomial_solver.py

Reply:
+1

Reply:
Hey! Thanks for reviewing it, I will move them to 'polynomial_solver.py'.